engine/views.php

```diff
@@ -11,7 +11,6 @@
 		 */
 		"jquery.js" => "vendor/bower-asset/jquery/dist/jquery.min.js",
 		"jquery.min.map" => "vendor/bower-asset/jquery/dist/jquery.min.map",
-		"jquery-migrate.js" => "vendor/bower-asset/jquery-migrate/jquery-migrate.min.js",
```

Comment:
why remove this as part of this PR?

Reply:
found a js lib which is no longer supplies